src/plugins/expressions/public/_expression_renderer.scss

```diff
@@ -12,8 +12,7 @@
   height: 100%;
 }
 
-.expExpressionRenderer-isEmpty,
-.expExpressionRenderer-hasError {
```

Comment:
Why is this selector being removed? We shouldn't be showing a blank chart alongside the error statement.

Reply:
The plan was to implement this a bit differently, but now this change is just reverted